basilisk/tests/test_edit_block_dialog.py

```
@@ -0,0 +1,69 @@
+from PIL import Image
+from upath import UPath
+
+from basilisk.conversation import (
+    Conversation,
+    Message,
+    MessageBlock,
+    MessageRoleEnum,
+    SystemMessage,
+)
+from basilisk.provider_ai_model import AIModelInfo
+from basilisk.gui.edit_block_dialog import EditBlockDialog
```

Ruff (0.11.9)

`basilisk.gui.edit_block_dialog.EditBlockDialog` imported but unused

Remove unused import: `basilisk.gui.edit_block_dialog.EditBlockDialog`

(F401)